### ISSUE TITLE
unable to set absolute db path

### ISSUE THREAD

**Issue by aion-chris** [aionnetwork/aion#627]
_Before filing a new issue, please **provide the following information**._

> I'm running:
>
> - **Which Aion version?**: 0.3.1.8dc68705f4c5c52008ea8a09074480fd9ce35fe3
> - **Which operating system?**: Linux
> - **Which distributor and version?**: Ubuntu 16.04
> - **How installed?**: via from source
> - **Are you fully synchronized?**: no
> - **Did you try to restart the node?**: yes

_Your issue description goes here below. Try to include **actual** vs. **expected behavior** and **steps to reproduce** the issue._
unable to absolute db path

config.xml 
```
<path>/some_absolute_db_path</path>
```
src
modDbImpl/src/org/aion/db/impl/DatabaseFactory.java -> connectBasic -> info.getProperty(Props.DB_PATH)

output
```
18-09-06 17:35:38.123 DEBUG DB   [main]: db factory: /home/chris/aion/some_absolute_db_path
```

**Comment by AlexandraRoatis** [comment]
This is not a bug, but a new feature request. Check the documented behavior: https://github.com/aionnetwork/aion/wiki/Database#database-location

You can create a symbolic link (`ln -s`) to the path you want.

**Comment by qoire** [comment]
@kzeine candidate for bounty

**Comment by AlexandraRoatis** [comment]
solved in #650
